src/main/java/com/google/api/codegen/transformer/SurfaceNamer.java

```diff
@@ -1027,4 +1027,12 @@ public String getApiCallableTypeName(ServiceMethodType serviceMethodType) {
   public String getReleaseAnnotation(ReleaseLevel releaseLevel) {
     return getNotImplementedString("SurfaceNamer.getReleaseAnnotation");
   }
+
+  public String getMetadataIdentifier() {
+    return getNotImplementedString("SurfaceNamer.getMetadataIdentifier");
+  }
+
+  public String getMetadataName() {
```

Comment:
I wonder if it would make sense to have a separate namer for metadata? Metadata files tend to be expressed in a different language from the repo language, and have mutually exclusive concepts. So, `PackageMetadataNamer`.